### PR TITLE
feat(comments): include pattern

### PR DESCRIPTION
**Pull request by kalinchernev**
Notes that I've changed patterns' status to ready for all that do not require further work on the js part.

### CODE REVIEW

**Comment by yhuard** [comment]
@degliwe what's our plan regarding the status of the components? When can we consider them as ready?

**Comment by degliwe** [comment]
They are "ready" only if we don't foresee any changes (do they comply to our standard?)

**Comment by yhuard** [comment]
Then none of them are...

File: framework/molecules/comments/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+dist
```
Comment (yhuard) [inline framework/molecules/comments/.gitignore]: Needed?

File: framework/molecules/breadcrumbs/breadcrumbs.config.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ module.exports = {
       _demo: {
         scripts: 'Europa.breadcrumbs();',
       },
+
```
Comment (yhuard) [inline framework/molecules/breadcrumbs/breadcrumbs.config.js]: Not needed 😄

File: .gitignore [hunk 3]
```diff
@@ -4,7 +4,6 @@
 # Ignore some folders
 node_modules
 /static/framework
-dist
```
Comment (yhuard) [inline .gitignore]: Wrong `.gitignore`, I was referring to `framework/molecules/comments/.gitignore` 😉

Comment (kalinchernev) [inline .gitignore]: Oh, sorry, I can push it back, but aren't all patterns supposed to have dist ignored on package-level? At least at the moment they all have it, so this change will be for all, no?

Comment (yhuard) [inline .gitignore]: The discussion is open. I'd rather have `dist` ignored in the root `.gitignore` than having a `.gitignore` per component. When we'll publish the components to npm, then we may reconsider it.

Comment (kalinchernev) [inline .gitignore]: So, for now I can remove this package-specific ignore rule for all, correct?

Comment (yhuard) [inline .gitignore]: For now, we should only have one `.gitignore` at the root of the project, with `dist` or `/dist`, I can accept both

Comment (kalinchernev) [inline .gitignore]: It's done

**Comment by yhuard** [comment]
Good for me, you can merge 🎉

**Pull request merged**